src/mds/CInode.cc

```diff
@@ -92,9 +92,9 @@ struct cinode_lock_info_t cinode_lock_info[] = {
   { CEPH_LOCK_IAUTH, CEPH_CAP_AUTH_EXCL },
   { CEPH_LOCK_ILINK, CEPH_CAP_LINK_EXCL },
   { CEPH_LOCK_IXATTR, CEPH_CAP_XATTR_EXCL },
-  { CEPH_LOCK_IFLOCK, CEPH_CAP_FLOCK_EXCL }  
+//{ CEPH_LOCK_IFLOCK, CEPH_CAP_FLOCK_EXCL }
```

Comment:
We should delete this rather than commenting it out...unless there's some reason to keep it around?